src/Core/Engine/Events/WindowResizeEvent.hpp

```
@@ -7,8 +7,8 @@ class WindowResizeEvent : public Event
 {
 public:
     GLFWwindow* Window;
-    int Width, Height;
+    double Width, Height;
 
-    WindowResizeEvent(GLFWwindow* window, int width, int height) : Window(window), Width(width), Height(height) {}
+    WindowResizeEvent(GLFWwindow* window, double width, double height) : Window(window), Width(width), Height(height) {}
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: Window

(cppcoreguidelines-pro-type-member-init)